OlinkAnalyze/R/read_npx_parquet.R

```diff
@@ -61,15 +61,18 @@ read_npx_parquet <- function (filename) {
   # If other platforms are to be reported as parquet too, we have to add
   # them to this array
   olink_platforms <- c("ExploreHT")
-  if (!(parquet_file$metadata$ProductType %in% olink_platforms)) {
+  if (!(parquet_file$metadata$Product %in% olink_platforms)) {
 
     stop("Only \"Olink Explore HT\" parquet files are currently supported.")
 
   }
 
 
   # Check if it is an NPX file
-  olink_files <- c("NPX File")
+  olink_files <- c("NPX File",
+                   "Extended NPX File",
```

Comment:
I dont think we (OA) are supporting Extended NPX or CLI exports, but I am ok if it is incidentally supported